Project.toml

```
@@ -1,6 +1,6 @@
 name = "Documenter"
 uuid = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
-version = "0.25.2"
+version = "0.25.3"
```

Let's bump this in a separate commit.
```suggestion
version = "0.25.2"
```